book/plugins/android/single-bundle.md

```diff
@@ -0,0 +1,35 @@
+## Single Bundle React Naitve
+
+![single-bundle.png](./single-bundle.png)
```

Comment:
this reference path does not exist

Reply:
<img width="1050" alt="Screenshot 2019-06-05 at 14 58 29" src="https://user-images.githubusercontent.com/31966073/58962129-7215e500-87a2-11e9-9cc8-da1fa73a4cc4.png">